05-html-css/css-framework.md

```diff
@@ -18,185 +16,144 @@ The usual components are:
-
-That means creating separate wireframes for the different ways a user may interact with your site. Ex: A phone vertically, a phone horizontally, a large tablet in either direction, a laptop, a large-screen  television, I could go on forever!
-
-In an effort to not overwhelm ourselves, we’ll only be concerned on three generic screen sizes: small, medium and large. We refer to the user's visible area of a web page as the **viewport**.
-
-Think of small as a very narrow viewport, like your phone horizontally. Medium when you are split screening your laptop screen with your browser on one half and another application on the other. And large as your browser full-screen on your laptop.
-
 ## Grid Layout
-A grid system is an invisible foundation of web page layout, which is used to divide the web page multiple grids.
+A grid system is an invisible foundation of web page layout, which is used to divide the web page multiple grids.  While CSS Grid has reduced the need for a framework grid system there are advantages to a grid system like Bootstrap or Foundation.  One nice advantage to Bootstrap's grid system is that it [can be easier to make grids responsive to varying screen sizes.](https://getbootstrap.com/docs/4.1/layout/grid/#responsive-classes)
 
 ![grid](imgs/7_Grid1-530x265.jpg)
 
-Grids have long been used in design to provide structure and balance for content in a layout. Interested in learning more about grid theory? [Read this](https://designschool.canva.com/blog/grid-design/)
-
+### Bootstrap's 12-column Flexbox-based grid system
+![Bootstrap Grid System](imgs/bootstrap-grid.jpg)
 
-### Foundation uses a 12 column grid.
-![Foundation Grid](http://foundation.zurb.com/assets/img/seo/feature-grid-1.png)
+At Ada we will continue to use CSS Grid & Flexbox for layout, but it's important to know that many sites use CSS framework layouts like the 12 column grid layout.  If you want to learn more about Bootstrap's Grid system you can check [this out](http://getbootstrap.com/docs/4.1/layout/grid/).
 
-At Ada we will continue to use CSS Grid & Flexbox for layout, but it's important to know that many sites use CSS framework layouts like the 12 column grid foundation layout.  If you want to learn more about Foundation's XY-Grid you can check [this out](https://foundation.zurb.com/sites/docs/xy-grid.html).  One nice advantage to Foundation's grid system is that it can be easier to make responsive to varying screen sizes.
+## Add Bootstrap to your project
 
-## Add Foundation to your project
-
- When choosing to use Foundation in a Rails project you can select to either:
+ When choosing to use Bootstrap in a Rails project you can select to either:
  1.  Manually copy and include all the CSS, JavaScript and SCSS files into your project.
- 2. Use a provided [foundation-rails](https://github.com/zurb/foundation-rails) gem.
+ 2. Use a provided [Bootstrap](https://github.com/twbs/bootstrap-rubygem) gem.
```

Comment:
I think that we probably don't need the detail in the rest of this section of adding the Bootstrap gem to the rails template -- Maybe a write up of that we did this on Monday's updated rails template!